src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -298,7 +298,7 @@ public async Task<WorkerStatus> GetWorkerStatusAsync()
                     await tcs.Task;
                     var elapsed = sw.GetElapsedTime();
                     workerStatus.Latency = elapsed;
-                    _workerChannelLogger.LogDebug($"[HostMonitor] Worker status request took {elapsed.TotalMilliseconds}ms");
+                    _workerChannelLogger.LogDebug("[HostMonitor] Worker status request took {totalMs}ms", elapsed.TotalMilliseconds);
```

Comment:
nit - space between the number and "ms" ?

Reply:
I think typically there isn't a space (see how we report duration in the screenshot I shared)